### PR TITLE
Address issue 164 in rancher

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: resources/content/schema/user/user-auth.json [hunk 1]
```diff
@@ -154,14 +154,14 @@
...
-        "registry": "cru",
-        "registry.serverAddress": "cr",
-
-        "registryCredential": "cru",
-        "registryCredential.publicValue": "cr",
-        "registryCredential.secretValue": "cr",
-        "registryCredential.storagePoolId": "cr",
-        "registryCredential.email": "cr",
+        "registry": "crud",
+        "registry.serverAddress": "cru",
```
Comment (ibuildthecloud) [inline resources/content/schema/user/user-auth.json]: not editable

File: resources/content/schema/user/user-auth.json [hunk 2]
```diff
@@ -154,14 +154,14 @@
...
+
+        "registryCredential": "crud",
+        "registryCredential.publicValue": "cru",
+        "registryCredential.secretValue": "cru",
+        "registryCredential.registryId": "cru",
```
Comment (ibuildthecloud) [inline resources/content/schema/user/user-auth.json]: not editable

File: resources/content/db/changelog.xml [hunk 3]
```diff
@@ -31,6 +31,5 @@
     <include file="db/core-024.xml"/>
     <include file="db/core-025.xml"/>
     <include file="db/core-026.xml"/>
-    <include file="db/core-027.xml"/>
```
Comment (ibuildthecloud) [inline resources/content/db/changelog.xml]: make a 028

**Comment by hibooboo2** [comment]
Tests are failing because you can't update a registryCredential without providing a registryId, i am not sure how to change this behavior.

File: tests/integration/cattletest/core/test_private_repos.py [hunk 4]
```diff
@@ -72,35 +72,35 @@ def _wait_until_stopped(container, admin_client, timeout=45):
 def _create_registry(client):
```
Comment (ibuildthecloud) [inline tests/integration/cattletest/core/test_private_repos.py]: change all tests to not reference super_client

**Pull request merged**